src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/runInPodNestedExplicitInherit.groovy

```diff
@@ -2,7 +2,7 @@ podTemplate(label: '$NAME-parent', containers: [
 		containerTemplate(name: 'golang', image: 'golang:1.6.3-alpine', ttyEnabled: true, command: '/bin/cat'),
 	]) {
 
-	podTemplate(inheritFrom: '-',  containers: [
+	podTemplate(inheritFrom: '<noInherit>',  containers: [
```

Comment:
No, this is still incompatible. Again, for compatibility, we must retain the meaning of no inheritance for `''`, and may only introduce a special value (not used at runtime) to mean _do_ inherit in the default manner.
```suggestion
	podTemplate(inheritFrom: '',  containers: [
```